admin-ui/app/components/bootstrap/bootstrap.html

```diff
@@ -18,29 +18,29 @@
   <div ng-if="bootstrap.types.android">
     <div class="form-group">
       <label class="col-sm-3 control-label" for="fcmApiKey"></label>
-      <div class="col-sm-7">Server Key<br>
+      <div class="col-sm-7"><span translate>Server Key</span><br>
         <input type="text" placeholder="e.g. AIza5a448c2f31700a466fbc9d33d33942b043a27596" id="fcmApiKey" class="form-control" ng-model="bootstrap.config.android.googleKey" ng-required="bootstrap.types.android">
       </div>
     </div>
     <div class="form-group">
       <label class="col-sm-3 control-label" for="fcmProjectNumber"></label>
       <div class="col-sm-7">
-        Sender ID<br>
+        <span translate>Sender ID</span><br>
         <input type="text" placeholder="e.g. 42" id="fcmProjectNumber" class="form-control" ng-model="bootstrap.config.android.projectNumber" ng-required="bootstrap.types.android">
       </div>
     </div>
   </div>
 
 
   <div class="form-group" ng-if="bootstrap.allowCreate('ios')">
-    <label class="col-sm-3 control-label sr-only">iOS</label>
+    <label class="col-sm-3 control-label sr-only" translate>iOS</label>
```

Comment:
I mean, does it make sense to translate "iOS"? Is like translating "Ubuntu" or "Javascript", no? Just wondering.